contracts/zero-ex/contracts/src/ZeroEx.sol

```diff
@@ -48,29 +51,52 @@ contract ZeroEx {
 
     /// @dev Forwards calls to the appropriate implementation contract.
     fallback() external payable {
```

Comment:
A plain `fallback() external payable` still creates some overhead in Solidity. We can avoid this if we use Yul directly.